test_utilities/bin/flutter_test_runner.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2019 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Runner for flutter tests. It expects a single parameter with the full
+# path to the flutter project where tests will be run.
+
+set -e
+
+echo "Running flutter tests from $1"
+pushd $1 > /dev/null
+
+flutter packages get
+# Remove the " || true" portion when https://github.com/flutter/flutter/issues/44370
+# is fixed.
+flutter analyze || true
+flutter config --enable-web
```

Comment:
Missing `flutter format --set-exit-if-changed lib/ test/` which ensures the code being committed followers the formatter

Reply:
Added!